types/validtransaction_bench_test.go

```diff
@@ -0,0 +1,59 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/NebulousLabs/Sia/crypto"
+	"github.com/NebulousLabs/Sia/encoding"
+)
+
+// BenchmarkStandaloneValid times how long it takes to verify a single
+// large transaction, with 100 signatures
+func BenchmarkStandaloneValid(b *testing.B) {
+	// make a transaction 10 with valid inputs with valid signatures
+	b.ReportAllocs()
+	txn := Transaction{}
+	numSigs := 100
```

Comment:
100 sigs is already more than what's allowed by transaction pool. But I agree that having 5 or so signatures would be more interesting to see normal usage.